app/logging/Logging.scala

```diff
@@ -20,5 +20,5 @@ import play.api.Logger
 
 trait Logging {
 
-  protected val logger: Logger = Logger(s"application.${this.getClass.getCanonicalName}")
+  protected val logger: Logger = Logger(s"logger.application.${this.getClass.getCanonicalName}")
```

Comment:
out of curiosity why the logger?

Reply:
It seems to be a a pattern that most services use. It did make sense to allow for all logging overrides to be grouped together in config